src/schema/datatypes/events.yaml

```diff
@@ -0,0 +1,15 @@
+group1:
+  datatypes:
+    - meg
+    - eeg
+    - ieeg
+  suffices:
```

Comment:
shouldn't it be `suffixes`?

Reply:
I feel like I have not done enough harm today, so now I am on it -- I will fix this and also rename `datatypes/` into `modalities/` since that is what it is! ;)

Reply:
no - I will stop doing harm -- I think it is a bigger question here, since it is neither datatypes nor modalities ATM. as for why it is not `datatypes/` -- because datatype shouldn't include another datatype (or itself), which it does now. I have pushed `suffices` -> `suffixes' though

Reply:
🤦 yes suffixes my bad. thanks for catching it!